### PR TITLE
Pin microerror version to v0.1.0

### PR DESCRIPTION
**Pull request by fiunchinho**
We should pin or dependency on microerror, since newer versions have breaking changes.

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 
 [[constraint]]
-  branch = "master"
+  version = "v0.1.0"
```
Comment (kopiczko) [inline Gopkg.toml]: I thought we just agreed on that?

```suggestion
  version = "~0.1.0"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.1] 2020-03-04
+
+### Changed
+
+- Updated microerror to `v0.1.0.
```
Comment (kopiczko) [inline CHANGELOG.md]: Can we make a release as a separate PR?

File: Gopkg.toml (hunk 1 above)
Comment (fiunchinho) [reply Gopkg.toml]: I'm stupid. Fixing it, sorry.

File: CHANGELOG.md (hunk 2 above)
Comment (fiunchinho) [reply CHANGELOG.md]: Ok

Comment (fiunchinho) [reply CHANGELOG.md]: https://github.com/giantswarm/micrologger/pull/37
